src/classifier/Pipe.cpp

```diff
@@ -449,18 +462,30 @@ void Pipe::Run() {
 void Pipe::ClassifyInstance(Instance *instance) {
   Parts *parts = CreateParts();
   Features *features = CreateFeatures();
+  std::vector<double> scores;
   std::vector<double> gold_outputs;
   std::vector<double> predicted_outputs;
-  std::vector<double> scores;
 
   Instance *formatted_instance = GetFormattedInstance(instance);
 
+  //Create parts for this instance
```

Comment:
// Create parts for this instance.

- add space after //
- end with punctuation.